content/articles/how-to-create-a-winforms-io-manager-for-removable-drives-in-c#/index.md

```diff
@@ -0,0 +1,425 @@
+
+Let's drag and drop two radio buttons into the groupbox and change their `text` properties to `Enable USB Ports` and `Disable USB Ports` :
+By now you should have something like this :
+
+![groupbox1](/engineering-education/how-to-create-a-winforms-io-manager-for-removable-drives-in-c#/groupbox1.jpg)
+
+#### 2.6 - Adding the Read/Write controls
+In this step, we will drag and drop another groupbox just below the first one(make sure you align them for a better design) then change its text property to `Read/Write`, and set its dimensions to `560, 150` under its size property.
+It is also advisable to match its background color with the other one(ActiveCaption). We will also go on and add two radio buttons into this groupbox and change their text properties to `Activate readonly for removable drives` and `Clear readonly for removable drives` respectively :
+
+![both radioButtons](/engineering-education/how-to-create-a-winforms-io-manager-for-removable-drives-in-c#/both_radiobuttons.jpg)
+
+#### 2.7 - Adding the control buttons
+Now what is left for the UI to be complete is the addition of the control buttons. For these, we shall drag and drop a button and place it just below the `Read/Write` groupbox. Then set the following properties for the button:
+-  Backcolor to `Black`
+- FlatStyle to `Popup` 
+- Font name to `Perpetua Titling MT` 
+- Font size to `12` 
+- Forecolor to `White`
+- it's size property to `100, 40`
+
+
+![button properties](/engineering-education/how-to-create-a-winforms-io-manager-for-removable-drives-in-c#/btn_properties.jpg)
+
+Now copy the button and paste it twice in a horizontal line(next to each other below the groupbox)
+Then edit their `Text` properties to `OK`, `Cancel` and `Close` respectively. Our final result should look like this :
+
+![with all buttons](/engineering-education/how-to-create-a-winforms-io-manager-for-removable-drives-in-c#/with_all_buttons.jpg)
+
+### Step 3 - Adding Code to the Form
+#### 3.1 Adding required namespaces
+To edit the code, right-click on the form(the teal background) then select `view code` and Visual Studio will load `Form1.cs`. Now we will add the namespaces commented `new` just below the default ones :
+```C#
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+using Microsoft.Win32;                  // new
+using System.Runtime.InteropServices;   // new
+``` 
+
+`Microsoft.Win32` - provides us with a class that handles events that manipulate the system registry.
+`System.Runtime.InteropServices` - supports COM interop and platform invoke services.
```

Comment:
These are hanging, are they explanations of the prev code? Work on your article flow! It's there throughout the article!